schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
I don't think this is particulary true only from `ha` it is a `sles` feature, you can drop that from the name:
`default_s390x_kvm_textmode.ymal`

Reply:
The schedule/yast/sle/flows/default_s390x_kvm_textmode.yaml already existed, the name of default_s390x_kvm_ha_textmode.yaml just means specific for ha_textmode test suite.

Reply:
then as @hjluo mentioned, we don't need to duplicate files, you can reuse it, those extra preparation test modules can be added to ha. You need verification for the test suite currently using it.

Reply:
Ok, reuse the schedule/yast/sle/flows/default_s390x_kvm_textmode.yaml